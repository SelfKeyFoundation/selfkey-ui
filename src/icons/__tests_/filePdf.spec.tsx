import * as React from 'react';
import { create } from 'react-test-renderer';
import FilePdfIcon from '../file-pdf';

describe('Filed Pdf Icon', () => {
  it('should show Filed Pdf Icon', () => {
    const component = create(
      <FilePdfIcon />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});