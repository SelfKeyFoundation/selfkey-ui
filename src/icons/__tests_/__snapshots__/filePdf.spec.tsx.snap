// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filed Pdf Icon should show Filed Pdf Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root Component-root-1"
  focusable="false"
  viewBox="0 0 29 36"
>
  <g
    fill="none"
    fillRule="evenodd"
    id="ðŸ’»-ID-Dashboard-Pdf-Icon"
    stroke="none"
    strokeWidth="1"
  >
    <g
      fill="#697C95"
      fillRule="nonzero"
      id="File-Pdf-Icon"
      transform="translate(-392.000000, -1401.000000)"
    >
      <g
        id="Area-box-3"
        transform="translate(150.000000, 1262.000000)"
      >
        <g
          id="Table"
          transform="translate(14.000000, 80.000000)"
        >
          <g
            id="Col2"
            transform="translate(228.000000, 11.000000)"
          >
            <g
              id="line"
              transform="translate(0.000000, 46.000000)"
            >
              <path
                d="M0,2 L0,38 L28.173913,38 L28.173913,37.2173913 L28.173913,12.4949047 L17.3610736,2 L0,2 Z M1.56521739,3.56521739 L16.4347826,3.56521739 L16.4347826,13.7391304 L26.6086957,13.7391304 L26.6086957,36.4347826 L1.56521739,36.4347826 L1.56521739,3.56521739 Z M18,4.80180025 L25.5968069,12.173913 L18,12.173913 L18,4.80180025 Z M13.1255092,15.9891304 C12.536741,15.9891304 11.9301417,16.3233677 11.6611756,16.8160669 C11.3922094,17.3087652 11.3766073,17.8389513 11.4425953,18.3889264 C11.5466165,19.2558886 11.9406076,20.239893 12.4514264,21.2426973 C12.1974574,22.1006423 12.0748132,22.8470052 11.6932743,23.7601906 C11.2072343,24.9234891 10.6115799,25.8483268 10.0378737,26.8172558 C9.29222922,27.1655871 8.3517973,27.477398 7.80163043,27.8383152 C7.1838,28.2436204 6.69614165,28.6095119 6.40913713,29.1864808 C6.26563487,29.4749652 6.19235922,29.8830299 6.33423913,30.2533963 C6.47554304,30.6222664 6.76820583,30.8642749 7.07099165,31.0222483 C7.71162417,31.3589102 8.45232417,31.1497001 8.98777174,30.788383 C9.52362078,30.426795 10.0002193,29.898668 10.4796196,29.2537366 C10.7223832,28.9271493 10.91484,28.3973443 11.1521739,28.0156254 C11.9180872,27.67511 12.4636923,27.3261182 13.3349181,27.0251355 C14.5123983,26.6183543 15.5611064,26.4487285 16.661005,26.2088997 C17.5823937,26.8309687 18.560934,27.3109716 19.6294156,27.3109716 C20.2345216,27.3109716 20.7030874,27.2855681 21.1594766,27.0373645 C21.6158666,26.7891625 21.8916438,26.1930816 21.8916438,25.7381112 C21.8916438,25.3674161 21.728293,24.9746913 21.4712977,24.7201087 C21.2143015,24.4655261 20.9068169,24.3319911 20.5939197,24.2477918 C19.9681263,24.0793948 19.2592174,24.0889598 18.4432743,24.1744223 C18.0108532,24.2197118 17.4347257,24.4468178 16.9544833,24.5382132 C16.8913041,24.4869602 16.8278103,24.4655245 16.7649457,24.4113453 C15.7885638,23.5698367 14.8762604,22.4077309 14.1985393,21.2533963 C14.1564193,21.1816468 14.1638752,21.135595 14.1236413,21.0638587 C14.2876988,20.445338 14.6129729,19.7291 14.6861413,19.186821 C14.7870383,18.4390392 14.8090477,17.7843025 14.6280569,17.1844431 C14.5375583,16.884513 14.3843752,16.5835178 14.1175268,16.3498645 C13.8506807,16.1162097 13.4762783,15.9891304 13.1255092,15.9891304 Z M13.0597826,17.5482341 C13.070095,17.5487678 13.082868,17.5543877 13.0979958,17.5574047 C13.101048,17.5622569 13.1068283,17.5598308 13.1300953,17.6368888 C13.1733344,17.7801883 13.1475678,18.1874219 13.1331522,18.5478938 C13.1144478,18.4547329 13.0070223,18.285039 12.9971129,18.2024457 C12.9533494,17.8377226 13.0114855,17.6074517 13.0337977,17.5665761 C13.0416159,17.5522504 13.0494702,17.5476996 13.0597826,17.5482341 Z M13.5336279,23.0387224 C14.0155317,23.7338065 14.5520257,24.3801161 15.1508152,24.9784306 C14.3484817,25.1921681 13.6314767,25.266692 12.8243888,25.5455167 C12.6541276,25.6043368 12.5539333,25.6849635 12.3856998,25.7457542 C12.6254003,25.2478609 12.9207365,24.8796787 13.136209,24.3639607 C13.3314323,23.8967097 13.3651706,23.5035286 13.5336279,23.0387224 Z"
                id="icon-file-pdf"
              />
            </g>
          </g>
        </g>
      </g>
    </g>
  </g>
</svg>
`;
